src/main/resources/wa-task-configuration-ia-asylum.dmn

```diff
@@ -300,6 +302,20 @@ else if (taskAttributes != null and taskAttributes.taskType != null and taskAttr
 else ""</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0fpew6w">
+        <inputEntry id="UnaryTests_1dsxa1j">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1ul72fr">
+          <text>"reviewSpecificAccessRequestJudiciary","reviewSpecificAccessRequestLegalOps","reviewSpecificAccessRequestAdmin"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_1f53slh">
+          <text>"workType"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0huw4e7">
+          <text>"access-requests"</text>
```

Comment:
Just checking this is an actual workType? maybe add the confluence doc showing this to the ticket?

Reply:
on the ticket